src/libraries/System.Security.Cryptography.Cng/tests/PropertyTests.cs

```diff
@@ -160,5 +160,31 @@ public static void NullValueRoundtrip()
             CngProperty property = new CngProperty("banana", null, CngPropertyOptions.CustomProperty);
             Assert.Null(property.GetValue());
         }
+
+        [Fact]
+        public static void TestConstructorSpan()
+        {
+            var name = "dotnet-test";
+            ReadOnlySpan<byte> value = [1, 2, 3, 4, 5];
+
+            CngProperty property = new CngProperty(name, value, CngPropertyOptions.CustomProperty);
+            AssertExtensions.SequenceEqual(value, property.GetValue());
+        }
+
+        [Fact]
+        public static void TestConstructorSpan_NameNull()
+        {
+            var name = "dotnet-test";
+            var value = new byte[12];
+            value[5] = 1;
+            value[6] = 2;
+            value[7] = 3;
+
+            Assert.Throws<ArgumentNullException>(() =>
+            {
+                ReadOnlySpan<byte> span = value;
+                _ = new CngProperty(name, span, CngPropertyOptions.CustomProperty);
+            });
```

Comment:
This assertion should be validating the name of the parameter that triggered the ArgumentNullException.  IIRC, something like

```suggestion
            Assert.Throws<ArgumentNullException>(() =>
            {
                ReadOnlySpan<byte> span = value;
                _ = new CngProperty(name, span, CngPropertyOptions.CustomProperty);
            },
            "name");
```

But it's not clear why this line is expected to throw anyways.  The test says "NameNull", but name is `"dotnet-test"`, so this looks like it should be not throwing.

Reply:
oh yes... my mistake